modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-multitenancy_{context}"]
+= Multi-tenancy
+
+Multi-tenancy with authentication and authorization is provided in the Tempo Gateway service.
+The authentication OppenShift OAuth and Kubernetes `TokenReview` API, the authorization uses Kubernetes `SubjectAccessReview`.
+
+.Sample Tempo CR with two tenants: dev and prod
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind:  TempoStack
+metadata:
+  name: simplest
+spec:
```

Comment:
Should we specify mandatory storage spec in the example.

Reply:
Other examples don't specify it https://docs.openshift.com/container-platform/4.13/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.html